data/en/test_submitted_responses.json

```diff
@@ -0,0 +1,113 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo radio field Other input.",
+    "view_submitted_responses": {
+        "enabled": true,
+        "timeout": 5
+    },
```

Comment:
I think we can improve the naming here. It would be clearer if we renamed:
- `view_submitted_responses` to `view_submitted_response` as it's for a single questionnaire response, and
- `timeout` to `duration`